app/core/drucker_dashboard_client.py

```diff
@@ -115,20 +116,19 @@ def run_switch_service_model_assignment(self, model_path:str):
             raise Exception(response["message"])
         return response
 
-    def __eval_model_request(self, f:FileStorage):
+    def __eval_model_request(self, f:FileStorage, data_path:str):
         for data in ProtobufUtil.stream_file(f):
-            request = drucker_pb2.EvaluateModelRequest(data=data)
+            request = drucker_pb2.EvaluateModelRequest(data=data, data_path=data_path)
             yield request
 
-    def run_evaluate_model(self, f:FileStorage):
+    def run_evaluate_model(self, f:FileStorage, data_path:str):
         try:
-            request_iterator = self.__eval_model_request(f)
-            response = self.stub.EvaluateModel(request_iterator)
-            response = ProtobufUtil.serialize_to_object(response)
-            return response
+            request_iterator = self.__eval_model_request(f, data_path)
+            return protobuf_to_dict(self.stub.EvaluateModel(request_iterator),
+                                    including_default_value_fields=True)
```

Comment:
I tried using `protobuf_to_dict` library because `ProtobufUtil.serialize_to_object` ignored default values and does not cover `map<string, float>` data type.

It is also proposal, so I used it only at this part, but If you agree with using the library, I will replace `ProtobufUtil.serialize_to_object` with `protobuf_to_dict`

Reply:
How do you think @yustoris ? You developed `serialize_to_object`.